base/uk.ac.stfc.isis.ibex.e4.client/plugin_customization.ini

```diff
@@ -8,12 +8,7 @@ org.eclipse.ui/KEY_CONFIGURATION_ID = IBEX_key_scheme
 org.csstudio.email/smtp_host=exchsmtp.stfc.ac.uk
 
 # Channel Access
-## for moment turn off auto address as it doesn't seem to work with current CSStudio
-org.csstudio.platform.libs.epics/auto_addr_list=false
-org.csstudio.platform.libs.epics/use_pure_java=true
-org.csstudio.platform.libs.epics/addr_list=127.255.255.255 130.246.51.255 130.246.55.255 130.246.39.255 130.246.59.255 130.246.39.152:5066
-org.csstudio.platform.libs.epics/max_array_bytes=65536
-org.csstudio.platform.libs.epics/conn_tmo=30.0
```

Comment:
These settings are needed for genie_python, otherwise you get the below message

![image](https://user-images.githubusercontent.com/24375813/45948477-0c0c7780-bff0-11e8-99ba-e6dcbd7cfe61.png)